### DISCUSSION TITLE
Removing some plugins

### DISCUSSION THREAD

**Discussion by l0drex** [oskarsh/Yin-Yang#193] [Polls]
I am thinking about removing some plugins, since I don't think many people use them. This includes:

- **Atom**, since its end of life
- **Gnome**, since there is [an extension](https://extensions.gnome.org/extension/2236/night-theme-switcher/) that integrates more nicely
- **Gedit**, since there is Gnome TextEditor
- **Wallpaper support for Gnome**, since the extension covers this
- **Sound** option
- **Firefox** plugin, as there is the system theme that syncs with the system

This is mostly because I don't know if and for how long they work, and I don't want to confuse anyone with a broken app. Please let me know what you think!

**Comment by l0drex** [comment]
I have now removed the Atom, Gedit, Brave and Sound plugins in the latest beta. I will keep the Firefox plugin for now, since it allows selecting themes. Gnome stuff will stay as well

**Comment by relative** [comment]
@l0drex what do you think about removing the vscode plugin? there's [a longstanding bug](https://github.com/oskarsh/Yin-Yang/issues/84) where your settings.json is completely overwrote if theres a comment or trailing comma at the end

**Reply by l0drex** [reply]
I guess the vscode plugin is very popular, so I would rather fix it. Originally I was hesitant to do so, because it would add another dependency and I hoped the functionality would be implemented in the app itself. 

Unfortunately I am really busy at the moment, so I cant really spend much time on this project. So it will probably stay as it is for at least half a year.
